.github/workflows/ci.yml

```diff
@@ -62,7 +62,13 @@ jobs:
     - name: Test (debug)
       run: cd build_dbg && ctest
     - name: Test (release)
-      run: cd build_dbg && ctest
+      run: cd build && ctest
+    - name: Package (deb)
+      run: cd build && cpack -G "DEB" ..
+    - uses: actions/upload-artifact@v2
+      with:
+        name: ubuntu_latest_build
```

Comment:
I'm not fond of the `ubuntu-latest` tag

Reply:
Maybe we should change this action to use the [ubuntu-18.04](https://github.com/actions/virtual-environments) environment instead of latest (assuming that gcc-8 can be installed without issues), or explicitly pin it to `20.04`. That seems better for CI stability in general. And then the artifact could be called accordingly.

Reply:
Maybe we should pin CI to 20.04 explicitly, then the artifact tag could be named accordingly?

(Re-added comment that disappeared)